src/movies.js

```diff
@@ -1,22 +1,81 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+
+function getAllDirectors(moviesArray) {
+    const trackMovies = [];
+    for (let i = 0; i < moviesArray.length; i++) {
+        const movie = moviesArray[i];
+        trackMovies.push(movie.director);        
+    }
+    return trackMovies;
+}
```

Comment:
Esto esta perfecto literalmente!
Te pongo como se haria usando el metodo map y me dices, acostumbrarnos a usar el metodo map es TOP :D
```suggestion
function getAllDirectors(moviesArray) {
    return moviesArray.map((movie) => movie.director);
}
```